### PR TITLE
feat: equipment availability

### PR DESCRIPTION
**Pull request by nubsthead**

### CODE REVIEW

**Review by FrancescoGottardi — DISMISSED** [review]

File: src/view/editor/parts/editor-equipment.tsx [hunk 1]
```diff
@@ -49,15 +76,75 @@ export const EditorEquipment = ({ editorId }: { editorId: string }): ReactElemen
...
+					end,
+					attendeesAvailabilityList,
+					allDay
+				);
+
+				if (isBusyAtTimeOfEvent) {
+					return {
+						..._equipment,
+						email: _equipment?.email ?? _equipment?.label,
+						customComponent: (
```
Comment (gnekoz) [inline src/view/editor/parts/editor-equipment.tsx:93]: IMO the custom component can be defined in a separated component.
In this way the new component can be used for the rooms availability too, a test for the component rendering will be easier and the current component readability will be improved

File: src/view/editor/parts/editor-equipment.tsx [hunk 2]
```diff
@@ -49,15 +76,75 @@ export const EditorEquipment = ({ editorId }: { editorId: string }): ReactElemen
...
+									<Row>
+										<Icon icon="AlertTriangle" color="error" />
+									</Row>
+								</Tooltip>
+							</Container>
+						)
+					};
+				}
+			}
+			return {
```
Comment (giuliano176) [inline src/view/editor/parts/editor-equipment.tsx:99]: for readability sake i would return this condition first

**Review by giuliano176 — DISMISSED** [review]

File: src/view/editor/parts/editor-equipment.tsx (hunk 2 above)
Comment (nubsthead) [reply src/view/editor/parts/editor-equipment.tsx:99]: This condition is covering 2 different cases and is placed at the end as return default value of the map.
It is covering the if (!isBusyAtTimeOfEvent) and the default, so i prefer it to be last or i have to duplicate this return for the first and last case

File: src/view/editor/parts/editor-equipment.tsx (hunk 1 above)
Comment (nubsthead) [reply src/view/editor/parts/editor-equipment.tsx:93]: Yes, you are right, we extracted the custom component, let me know if it is better now

**Review by FrancescoGottardi — APPROVED** [review]

**Review by gnekoz — APPROVED** [review]

**Pull request merged**